src/lib/linux-api/src/lib.rs

```diff
@@ -5,9 +5,24 @@
 //! libc headers, to work with these kernel types. Normally this can't be done
 //! because the libc and kernel headers contain incompatible definitions.
 //!
-//! To support this use-case, we prefix most names with `Linux` or `linux_`
+//! # Type names
+//!
+//! To support the re-exported C bindings, we prefix most names with `Linux` or `linux_`
 //! as appropriate, so that the re-exported types don't conflict with the libc
-//! types that would otherwise have the same names.
+//! types that would otherwise have the same names. These types should be used in C
+//! APIs to avoid name conflicts.
```

Comment:
Maybe expand on this part. If you use for example `internet::sockaddr_in` in your code and expose that to C, won't that show up as just `sockaddr_in` in the C bindings? If you want to expose it to C, wouldn't you have to cast it to a `bindings::linux_sockaddr_in` first so that cbindgen will expose it as `linux_sockaddr_in` instead?